CODEOWNERS

```diff
@@ -1,12 +1,12 @@
 # Technical writers
-README.*                                                      @Jimbo4350 @newhoggy @carbolymer @smelc @palas @olgahryniuk
-cardano-cli/README.md                                         @Jimbo4350 @newhoggy @carbolymer @smelc @palas @olgahryniuk
+README.*                                                      @Jimbo4350 @newhoggy @carbolymer @palas @olgahryniuk @CarlosLopezDeLara
```

Comment:
`olgahryniuk` does not have effect here, because she's not in any of the groups with write access. We should avoid adding users separately to repo access control list. So I think we should do either:
1. Remove Olga from the list
2. Copy the group [`docs-access`](https://github.com/orgs/input-output-hk/teams/docs-access) from iohk to I-mbo, and add the group here instead of you and Olga